auth/src/express/middleware/parser.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response } from 'express';
+
+import { Logger } from '../../api/services/logger';
+
+export namespace Parser {
+    function isValidJsonFormat(input: any): boolean {
+        return typeof input === 'string' && (input.trim().startsWith('{') || input.trim().startsWith('['));
+    }
+
+    export function parseJson(req: Request, res: Response, next: NextFunction): void {
+        const body = req.body;
+        Logger.debug('Trying to parse the request.body: ', body);
+        if (!body || !isValidJsonFormat(body)) {
+            req.body = {};
+        } else {
+            try {
+                req.body = JSON.parse(body);
+            } catch (e) {
+                Logger.error('Cannot read request.body: ', e);
```

Comment:
At this point you should return a 400er. Why: In `isValidJsonFormat` you checked that it is a string and it is (if it is valid json) an object or array. If this is true and the decoding fails, this is an invalid user input which should not be silenty ignored.